### DISCUSSION TITLE
How can I use Multiple Select so that the select box does not get too big?

### DISCUSSION THREAD

**Discussion by gonta1026** [JedWatson/react-select#5613] [Q&A]
## issue

When using Multiple Select as shown below, the component grows larger with each option that is selected. Is there any way to prevent this from becoming larger?

![スクリーンショット 2023-04-25 13 19 46](https://user-images.githubusercontent.com/45672476/234173601-7071796a-1f20-494b-8a58-0494d1567ed3.png)

**Comment by manjushsh** [comment]
Hi @gonta1026, You can limit height of control as shown in [this Code Sandbox](https://codesandbox.io/s/fixed-height-multiselect-react-select-0zd1y4)
You will need to scroll to see selected options.
You can find more details of styling in this [Docs](https://react-select.com/styles#inner-components)

**Reply by gonta1026** [reply]
@manjushsh 
Thanks for the sandbox and even the detailed documentation.
my issue has been resolved, I will close it.
